zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -25,7 +25,9 @@ import (
 	"github.com/zeta-chain/zetacore/zetaclient/zetacore"
 )
 
-// WatchInbound watches Bitcoin chain for incoming txs and post votes to zetacore
+// WatchInbound watches Bitcoin chain for inbounds
+// It starts a ticker and run ObserveInbound
+// TODO(revamp): move all ticker related methods in the same file
```

Comment:
to be explicit, maybe `move all ticker related methods to a separate file, e.g. tickers.go`

Reply:
Yeah, I was thinking the name of the file to be already evident, it can also be rediscussed, we could have something like `watchers.go`
The TODO(revamp) is more to have a mark to track everything that need to be considered